manual_flagging/yaml/inputs.yaml

```diff
@@ -13,7 +13,15 @@ run_note: |
 # without flags, it assigns a version number of "1". The latest
 # version of each sale flag can be found in the "default"."vw_pin_sale" view.
 
-manual_update: true
+manual_update: false
```

Comment:
**[Nitpick, non-blocking]** While we're here, can we remove the stray space that separates this key from the comment above it 😅